code/modules/keybindings/bindings_client.dm

```diff
@@ -2,7 +2,7 @@
 // These verbs are called for all key press and release events
 
 GLOBAL_LIST_INIT(valid_keys, list(
-	"F1" = 1, "F2" = 1, "F3" = 1, "F4" = 1, "F5" = 1, "F6" = 1, "F7" = 1, "F8" = 1, "F9" = 1, "F10" = 1, "F11" = 1, "F12" = 1,
+	"F1" = 1, "F2" = 1, "F3" = 1, "F4" = 1, "F5" = 1,"," = 1, "'" = 1, ";" = 1, "#" = 1, "F6" = 1, "F7" = 1, "F8" = 1, "F9" = 1, "F10" = 1, "F11" = 1, "F12" = 1,
```

Comment:
Why did you just randomly stick it between all the F-keys

Reply:
dunno just did

Reply:
> Is this tested?

compiling and testing soon

Reply:
Please move them